js/script.js

```diff
@@ -0,0 +1,55 @@
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+function renderBooks() {
+  const listDisplay = document.getElementById('list');
+  listDisplay.innerHTML = '';
+
+  books.forEach((book, index) => {
+    const bookDiv = document.createElement('div');
+    bookDiv.className = 'book';
+    const spanTitle = document.createElement('span');
+    spanTitle.textContent = book.title;
+    const spanAuthor = document.createElement('span');
+    spanAuthor.textContent = book.author;
+
+    const removeBtn = document.createElement('button');
+    removeBtn.className = 'btn';
+    removeBtn.textContent = 'Remove';
+    /* eslint-disable */
+    removeBtn.addEventListener('click', () => removeBook(index));
+    /* eslint-enable */
+    bookDiv.appendChild(spanTitle);
+    bookDiv.appendChild(spanAuthor);
+    bookDiv.appendChild(removeBtn);
+
+    listDisplay.appendChild(bookDiv);
+  });
+}
+
+function addBook() {
+  const titleInput = document.getElementById('title');
+  const authorInput = document.getElementById('author');
+  const title = titleInput.value;
+  const author = authorInput.value;
+
+  if (title && author) {
+    const bookData = {
+      title, author,
+    };
+    books.push(bookData);
```

Comment:
- [OPTIONAL] I'd like to suggest considering the `concat` method instead of the `push` method for adding elements. While both achieve the same goal of adding items to an array, `concat` returns a new array that includes the added elements, preserving the original array's integrity. This can help prevent unintended side effects and make your code more predictable. Give it a try and see how it enhances the clarity and maintainability of your code. Keep up the coding momentum!